Configuration/NodeTypes.FormPage.yaml

```
@@ -1,5 +1,5 @@
 'Neos.Form.Builder:FormPage':
-  label: "${String.cropAtWord(String.trim(String.stripTags(String.pregReplace(q(node).property('title') || q(node).property('label') || ((I18n.translate(node.nodeType.label) || node.nodeType.name) + (node.autoCreated ? ' (' + node.name + ')' : '')), '/<br\\W*?\\/?>|\\x{00a0}|[^[:print:]]|\\s+/u', ' '))), 100, '...')}"
+  label: "${String.cropAtWord(String.trim(String.stripTags(String.pregReplace(q(node).property('title') || q(node).property('label') || ((I18n.translate(node.nodeType.label) || node.nodeTypeName.value) + (node.autoCreated ? ' (' + node.name + ')' : '')), '/<br\\W*?\\/?>|\\x{00a0}|[^[:print:]]|\\s+/u', ' '))), 100, '...')}"
```

`.value` is not necessary anymore in fusion.

```suggestion
  label: "${String.cropAtWord(String.trim(String.stripTags(String.pregReplace(q(node).property('title') || q(node).property('label') || ((I18n.translate(node.nodeType.label) || node.nodeTypeName) + (node.autoCreated ? ' (' + node.name + ')' : '')), '/<br\\W*?\\/?>|\\x{00a0}|[^[:print:]]|\\s+/u', ' '))), 100, '...')}"
```

---

Seems to be not safe without the .value so we keep it.

---

What does "not safe" mean in this context?

---

I had a case where no explicit string type casting was present. Than it tried to set the NodeAggregateId object instead of the string representation.

E.g: https://github.com/Format-D/form-builder/pull/1/files#diff-84dbc8f6b25d214fbb5821fdfcffc738941395addf2d0181c58922e364c2da52R4

Without the `.value` this change above doesn't work.

---

But in all places in here, we have the string casting in place, right?
(I'm just curious whether I miss something. I'm totally fine with keeping the `.value` here!)

---

True, true. Was just a "better safe than sorry". I didn't wanted to test all cases. So I thought we simply keep it as it is.